src/lib/holocene/table/table-row.svelte

```diff
@@ -2,26 +2,15 @@
   import type { HTMLAttributes } from 'svelte/elements';
 
   interface $$Props extends HTMLAttributes<HTMLTableRowElement> {
```

Comment:
Looks like something changed with the table row truncation

| Before | After |
|--|--|
|<img width="1280" alt="Screenshot 2023-08-31 at 9 20 44 AM" src="https://github.com/temporalio/ui/assets/15069288/812a1e91-7689-46ad-9c53-89a6f744f84d">|<img width="1286" alt="Screenshot 2023-08-31 at 9 22 33 AM" src="https://github.com/temporalio/ui/assets/15069288/0e451015-0b6a-4a98-9f54-48fc202decf4">|